.editorconfig

```diff
@@ -0,0 +1,37 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+
+# Use 2 spaces for the HTML files
+[*.html]
+indent_size = 2
+
+# The JSON files contain newlines inconsistently
+[*.json]
+indent_size = 2
+insert_final_newline = sometimes
+
+# These JS files use tabs instead of spaces
+[**/admin/js/jquery.js]
```

Comment:
As mentioned somewhere else, let's move `jquery` to `vendor` directory and exclude whole directory.